mint.json

```diff
@@ -40,35 +40,20 @@
       "pages": ["welcome", "quickstart", "philosophy"]
     },
     {
-      "group": "Chapter 1 Aggregation and Collect",
-      "pages": [
-        "overview",
-        "csv-upload",
-        "survey",
-        "widget",
-        "public-portal",
-        "chrome-extension"
-      ]
+      "group": "Aggregation and Collection",
+      "pages": ["overview", "csv-upload", "survey", "widget", "public-portal", "chrome-extension"]
     },
     {
-      "group": "Chapter 2 Records",
+      "group": "Feedback Records",
       "pages": ["snippets", "contacts", "review-spam"]
     },
     {
-      "group": "Chapter 3 Feedback insights",
```

Comment:
I'm confused... I don't see "Chapter N" in live docs, or in my local, so surprised to see it showing them removed here. Maybe need to pull main locally? But, I think the main showing here on the left would align with the actual main...


![CleanShot 2024-08-27 at 12 22 53](https://github.com/user-attachments/assets/fc51a8e6-03b2-4a99-a49e-86c7c9b5b4d1)

Reply:
OH, I didn't notice the conflict below. Yeah, I think your main (mainly, the mint json file) was out of date!

Reply:
yep i re pulled and merged and seemed to get good!